float32[] p_targets
float32[] v_targets
float32[] p_gains
float32[] v_gains
float32[] ff_torque